### PR TITLE
Adding support to differentiate between home and shared assets

### PR DESCRIPTION
**Pull request by Robvdhout**

### CODE REVIEW

File: .github/repo-template/revision [hunk 1]
```diff
@@ -1 +1 @@
-cf8fce9f0a93e228a39c18ae9afd8700205a84c3
+eab101be40bd19ac0b016bc0106c9257027073b0
```
Comment (thijsputman) [inline .github/repo-template/revision:1]: Can you revert this?

It doesn't appear you actually applied any of the changes coming from https://github.com/studyportals/repo-template-php/pull/8 (which this change in the `revision`-file indicates)...

Comment (thijsputman) [reply .github/repo-template/revision:1]: (I'll draft a separate pull-request for that)

**Review by thijsputman — CHANGES_REQUESTED** [review]
What are "shared" assets?

Wouldn't it be better to call them "global" assets?

File: .github/repo-template/revision (hunk 1 above)
Comment (Alfalfamale) [inline .github/repo-template/revision:1]:
```suggestion
cf8fce9f0a93e228a39c18ae9afd8700205a84c3
```

**Comment by Alfalfamale** [comment]
> What are "shared" assets?
> 
> Wouldn't it be better to call them "global" assets?

Assets that need to be on every page. Maybe Site assets?

**Comment by thijsputman** [comment]
Not sure if putting "Site assets inside a Site" is the optimal way of describing things...

What's wrong with a `IGlobalAssetProviderProvider`?

**Pull request closed without merging**